thirdparty/fetch-thirdparty-deps.ps1

```diff
@@ -130,4 +130,13 @@ if (!(Test-Path "Eluant.dll"))
 	(New-Object System.Net.WebClient).DownloadFile("https://github.com/OpenRA/Eluant/releases/download/20140425/Eluant.dll", $target)
 }
 
+if (!(Test-Path "GeoLite2-Country.mmdb.gz"))
```

Comment:
http://stackoverflow.com/questions/16613656/how-can-i-check-if-a-file-is-older-than-a-certain-time-with-powershell